src/js/components/Chart/stories/Scan.js

```diff
@@ -123,5 +123,5 @@ const ScanChart = props => {
 };
 
 storiesOf('Chart', module).add('Scan', () => (
-  <ScanChart data={generateData(30, 100)} max={100} />
+  <ScanChart data={generateData()(30, 100)} max={100} />
```

Comment:
I wondering if there is a simpler way to use generateData, it feels a little verbose.